tests/run_cpu_tests.sh

```diff
@@ -2,8 +2,9 @@
 
 set -xeu
 
-CUDA_VISIBLE_DEVICES="" py.test --tx 4*popen//python=python$TRAVIS_PYTHON_VERSION --cov ignite --cov-report term-missing -vvv tests/
+CUDA_VISIBLE_DEVICES="" py.test --tx 4*popen//python=python$CI_PYTHON_VERSION --cov ignite --cov-report term-missing -vvv tests/
 
 export WORLD_SIZE=2
 
-CUDA_VISIBLE_DEVICES="" py.test --cov ignite --cov-append --cov-report term-missing --dist=each --tx $WORLD_SIZE*popen//python=python$TRAVIS_PYTHON_VERSION tests -m distributed -vvv
+# Rerun failing tests up to five times with a 30s delay
```

Comment:
Please remove reruns comment and args `--reruns 5 --reruns-delay 30`. Otherwise, good to be merged :)

Reply:
Whoops! Good catch.